tomviz/python/tomviz/_realtime/logger.py

```diff
@@ -41,18 +46,26 @@ def CHANGE_appendAll(self):
 
                 self.logTiltAngles = np.append(self.logTiltAngles, newAngle)
 
+                # Invert Contrast for BF-TEM
+                if self.invert:
+                    newProj *= -1
+
                 newProj = self.background_subtract(newProj)
-                newProj = self.center_of_mass_align(newProj)
+
+                # Apply Center of Mass (if selected)
+                if self.alignMethod == 'CoM':
+                    newProj = self.center_of_mass_align(newProj)
 
                 # Account for Python's disdain for AxAx1 arrays
                 # (compresses to 2D)
-                if(len(self.logTiltSeries) == 0):
+                if(len(self.logTiltSeries0) == 0):

```

Comment:
Whats `self.logTiltSeries` vs `self.logTiltSeries0`?

Reply:
@cjh1  I want a temporary copy of the tilt series prior to performing any alignments. When I only implemented center of mass this wasn't an issue since it's one step calculation. However for cross-correlation I'll need to iteratively apply this operation every time new data is read. I'm concerned the tilt series will diverge into weird local minima if we keep applying new alignment rather than performing cross-correlation once on the unaligned dataset (tiltSeries0).

Reply:
Ok, thanks